docs/userGuide/syntax/mathformulae.md

```diff
@@ -17,27 +22,36 @@ Solve the following simultaneous equations:
 
 \[\frac{2x}{3} + \frac{2y}{3} = 8\] (2)
 
-Euler's equation \( e^{i\pi}+1=0 \) is a beautiful equation.
+Euler's equation \(e^{i\pi}+1=0\) is a beautiful equation.
+
+$$ 4x + 5y = 16 $$ (3)
+
+$$\frac{10x}{3} + \frac{5y}{3} = 8$$ (4)
+
+Finally, the Pythagoras theorem: $c^2 = a^2 + b^2$.
 
 </variable>
 </include>
 
 <box type="info">
 
-If you would like to use <tooltip content="$ ... $ or $$ ... $$">dollars</tooltip> as the delimiter, you will need to enable the [texWithDollars](../usingPlugins.md#plugin-texwithdollars) plugin. 
-
-```markdown
-$2_3$
+Additional delimiters are possible with the [mathDelimiters](../usingPlugins.md#plugin-mathdelimiters) plugin.
```

Comment:
The box is a little large. Maybe

```
<modal>
<include src="that segment of the page">
</modal>
```

Reply:
I decided to remove it since the info is not very important and the link is provided for further reading if required.